v2g_liberty.py

```diff
@@ -74,13 +84,22 @@ def initialize(self):
         self.date_reference = datetime(2000, 1, 1)
         self.charger_in_error_since = self.date_reference
 
+        # For handling no_schedule errors
+        self.no_schedule_errors = {
+            "invalid_schedule": False,
+            "timeouts_on_schedule": False,
+            "no_communication_with_fm": False
+        }
+        self.notification_timer_handle = None
+        self.no_schedule_notification_is_planned = False
+
         self.client = self.configure_charger_client()
         self.log_errors()
         self.get_app("flexmeasures-client").authenticate_with_fm()
 
         self.listen_state(self.update_charge_mode, "input_select.charge_mode", attribute="all")
         self.listen_state(self.handle_charger_state_change, "sensor.charger_charger_state", attribute="all")
-        self.listen_event(self.restart_charger, "RESTART_CHARGER")
+        self.listen_event(self.run_test_code, "RESTART_CHARGER")
```

Comment:
This is now running test code?

Reply:
Remove

Reply:
The line originally called `self.restart_charger`. That's no longer needed?

Reply:
Indeed, this code did not really serve a purpose. Restarting the charger is needed when the communication via modbus with the charger is not possible. As this restart function is using modbus it was not useful. I'll also remove the restart button from the dashboard.